docs/concepts.rst

```diff
@@ -162,6 +162,32 @@ Example DAG with decorated style
         html_content=email_info['body']
     )
 
+DAG decorator
+-------------
+
+.. versionadded:: 2.0.0
+
+In addition to creating DAGs using context managed, in Airflow 2.0 you can also create DAGs from a function.
+DAG decorator creates a DAG generator function. This function when called returns a DAG.
+
+DAG decorator also sets up the parameters you have in the function as DAG params. This allows you to parameterize
+your DAGs and set the parameters when triggering the DAG manually. See
+:ref:`Passing Parameters when triggering dags <dagrun:parameters>` to learn how to pass parameters when triggering DAGs.
+
+You can also use the parameters on jinja templates by using the ``{{context.params}}`` dictionary.
+
+Example DAG with decorator:
+
+.. exampleinclude:: /../airflow/example_dags/example_dag_decorator.py
+    :language: python
+    :start-after: [START dag_decorator_usage]
+    :end-before: [END dag_decorator_usage]
+
+.. note:: Note that Airflow will only load DAGs that appear in ``globals()``. Which means you need to make sure to have
```

Comment:
```suggestion
.. note::

  Note that Airflow will only load DAGs that appear in ``globals()``. Which means you need to make sure to have
```

Reply:
(possibly need 4 spaces for the note block, not sure.)

Reply:
Looks good, downloaded artifact from the build-docs job 
![image](https://user-images.githubusercontent.com/8811558/97052477-fc6a0f00-1578-11eb-8e0b-8c7280aea321.png)